dev/squidex/schemas/teams.json

```diff
@@ -108,6 +108,21 @@
           "placeholder": "",
           "isRequired": false
         }
+      },
```

Comment:
Do we need this?

Reply:
Either the `fix:format` or squidex CLI's changes 🤷